st2tests/integration/mistral/test_filters.py

```diff
@@ -0,0 +1,373 @@
+        self.assertEqual(yaql_dict["title"], breaking_str)
+
+
+class RegexMatchFiltersTest(base.TestWorkflowExecution):
+
+    def test_regex_match(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_match',
+                                           parameters={"input_str": REGEX_SEARCH_STR_2})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertTrue(execution.result['result_jinja'])
+        self.assertTrue(execution.result['result_yaql'])
+
+    def test_regex_nomatch(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_match',
+                                           parameters={"input_str": REGEX_SEARCH_STR})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertFalse(execution.result['result_jinja'])
+        self.assertFalse(execution.result['result_yaql'])
+
+
+class RegexReplaceFiltersTest(base.TestWorkflowExecution):
+
+    def test_regex_replace(self):
+        execution = self._execute_workflow(
+            'examples.mistral-customfilters-regex_replace',
+            parameters={
+                "input_str": REGEX_SEARCH_STR_2,
+                "replacement_str": "foo"
+            }
+        )
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertEqual(
+            execution.result['result_jinja'],
+            "foo is your address. My address is foo."
+        )
+        self.assertEqual(execution.result['result_yaql'], "foo is your address. My address is foo.")
+
+
+class RegexSearchFiltersTest(base.TestWorkflowExecution):
+
+    def test_regex_search(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_search',
+                                           parameters={"input_str": REGEX_SEARCH_STR})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertTrue(execution.result['result_jinja'])
+        self.assertTrue(execution.result['result_yaql'])
+
+    def test_regex_nosearch(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_search',
+                                           parameters={"input_str": REGEX_SEARCH_STR_3})
+        execution = self._wait_for_completion(execution)
+        self._assert_success(execution, num_tasks=1)
+        self.assertFalse(execution.result['result_jinja'])
+        self.assertFalse(execution.result['result_yaql'])
+
+
+class RegexSubstringFiltersTest(base.TestWorkflowExecution):
+
+    def test_regex_substring(self):
+        execution = self._execute_workflow('examples.mistral-customfilters-regex_substring',
+                                           parameters={"input_str": REGEX_SEARCH_STR})
```

Comment:
Can you move the regex pattern from vars to input parameters here?  It was hard to follow at first why the output of the workflow returns "567 Elsewhere Dr"?